Global/show-char-code.ini

```diff
@@ -0,0 +1,64 @@
+[Command]
+Name=Show Character Code
+Command="
+    copyq:
+    var maxChars = 18;
+    
+    function padStart(str, len, c) {
+        str = str ? str.toString() : '';
+    
+        if (str.length >= len) {
+            return str;
+        }
+    
+        var rpt = c || ' ';
+        for (var i = 1; i < len - str.length; i++) {
+            rpt += c;
+        }
+    
+        return rpt + str;
+    }
+    
+    function getCodeInfo(code) {
+        var chunks = [
+            'U+' + padStart(code.toString(16).toUpperCase(), 4, '0'),
+            '&#x' + code.toString(16) + ';',
+            code.toString(16),
+            '&#' + code + ';',
+            code,
+        ];
+    
+        return chunks.join(' | ');
+    }
+    
+    // # main
+    
+    var text = str(input()) || str(clipboard());
+    
+    do {
+        var charData = [];
+    
+        for (var i = 0, max = Math.min(maxChars, text.length); i < max; i++) {
+            var c = text[i];
+            charData.push(c);
+            charData.push(getCodeInfo(c.charCodeAt(0)));
+        }
+    
+        var options = dialog.apply(this,
+            [
+            '.title', 'Show Character Code',
+            '.label', 'Shows Unicode code info for the first ' + maxChars + ' characters of the given text.'
+                + '<br>Info format: &lt;unicodeNotation&gt; | &lt;xmlRefHex&gt; | &lt;hexCode&gt; | &lt;xmlRefDec&gt; | &lt;decCode&gt;',
+            'Text', text
```

Comment:
You can add following arguments to use monospace font in the dialog so the layout is more uniform:

```
    '.style', 'font-family: monospace',
```

...or use `'<tt>' + c + '</tt>'` (in the loop above) to make the label sizes same (for most characters).

Reply:
Hi, thanks for this note - as you have already merged this PR, I continue with this idea of yours at /pull/30.